pkgs/servers/sql/postgresql/ext/pgml/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, buildPgxExtension
+, fetchFromGitHub
+, nix-update-script
+, python3
+, pkg-config
+, openssl
+, cmake
+, postgresql
+, cargo-pgx_0_7_1
+}:
+let
+  pname = "postgresml";
+  version = "2.4.0";
+  src = fetchFromGitHub {
+    owner = "postgresml";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-jTdMqhRNGnKnIdek6/IKyWZ/Wtt/5hTfaEMk72agGIc=";
+    fetchSubmodules = true;
+  };
+in
+(buildPgxExtension.override {cargo-pgx = cargo-pgx_0_7_1;}) {
+  inherit pname version postgresql;
+
+  src = "${src}/pgml-extension";
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "lightgbm-0.2.3" = "sha256-3rTO5gfbuBTN1S/9v/YvBJ9kHcl/GIL4HOipmbBWdFg=";
+      "xgboost-0.2.0" = "sha256-FejUNUtVHx+I0bYrz6aPiEvejGucmByE1NWGktwpHes=";
+    };
+  };
+
+  nativeBuildInputs = [
+    python3
+    pkg-config
+    cmake
+  ];
+
+  # Needed to get openssl-sys to use pkg-config.
+  env.OPENSSL_NO_VENDOR = 1;
+
+  buildInputs = [ openssl ];
+
+  passthru = {
+    updateScript = nix-update-script { };
+  };
+
+  meta = with lib; {
+    longDescription = "An is an end-to-end machine learning system. It enables you to train models and make online predictions using only SQL, without your data ever leaving your favorite database. ";
```

Comment:
1. multiline this
2. short description too

Reply:
Thanks!